### PR TITLE
Deactivate failing Tike test notebook priority 1

### PR DESCRIPTION
**Pull request by jaytmiller**
Without the installation of eleanor or other similar changes,  the priority 1 notebook for Tike is not going to pass.   Deactivate for now.

### CODE REVIEW

File: tools/image-build [hunk 1]
```diff
@@ -33,22 +33,27 @@ image-update
...
+image-base
 
 
 # ..................................................................................
 # Build STScI shared layer for all deployments
 cd ${COMMON_DIR}
 echo "========================= Building Common Image ========================"
-time docker build --tag ${COMMON_ID} --build-arg USE_FROZEN=${USE_FROZEN} .
-
+time docker build --progress plain --tag ${COMMON_ID} \
```
Comment (cslocum) [inline tools/image-build:43]: Does adding `--progress` slow down the build at all?  Probably not, just wondering.

**Review by cslocum — APPROVED** [review]
This looks good to me!  Merge away.  Did leave one question.

File: tools/image-build (hunk 1 above)
Comment (jaytmiller) [reply tools/image-build:43]: Not that I noticed.

Comment (jaytmiller) [reply tools/image-build:43]: We should be able to verify the timing once we get working builds again.

**Pull request merged**